tableofcontents.md

```diff
@@ -22,18 +22,25 @@ Run the following R code to load {MathEpiDrill} package on the working space.
 Do some calculation using basic operators in R, for example, +, -, *, /, %/%, ^, etc.
 
 ### Math functions
-Try some math functions, for example, exponential, trigonometric functions, and others.
+Try some math functions, for example, exponential, trigonomaetric functions, and others.
+Please see some paper that uses exponential distribution to estimate something (Ideas to be updated)
```

Comment:
Maybe move to the estimation section? This section is only to let them use some basic R syntax so estimation is too much here